app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -1,72 +1,75 @@
 /* @flow */
 
-import React from 'react'
-import classnames from 'classnames'
+import React from "react";
+import classnames from "classnames";
 
-import {
-  Body,
-  Caption,
-  Title,
-} from '../'
+import { Body, Caption, Title } from "../";
 
 type CurrencyProps = {
-  align?: 'left' | 'center' | 'right',
+  align?: "left" | "center" | "right",
   className: String,
-  currencySymbol: '$',
+  currencySymbol: "$",
   label?: String,
-  separator?: '.' | ',',
-  size?: 'sm' | 'lg',
+  separator?: "." | ",",
+  size?: "sm" | "lg",
   value?: String,
-  unit?: '00',
-}
+  unit?: "00"
```

Comment:
Same as above for `currentSymbol`. I think the type here should be String (`unit?: String`)